jax/__init__.py

```diff
@@ -17,6 +17,11 @@
 _os.environ.setdefault('TF_CPP_MIN_LOG_LEVEL', '1')
 del _os
 
+# Set Cloud TPU env vars if necessary before transitively loading C++ backend
+from .cloud_tpu_init import cloud_tpu_init as _cloud_tpu_init
+_cloud_tpu_init()
```

Comment:
Would it be worth wrapping this in a try/except? I'm worried this could make jax unimportable if something unanticipated goes wrong.

Reply:
Good idea, done. I made it raise a warning with the exception, because in theory this should never fail (like so much of jax!). Lemme know if you think it should be silent instead.

Reply:
Warning looks great 👍